Code/Framework/AzCore/AzCore/Math/MathStringConversions.h

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#pragma once
+
+#include <AzCore/std/functional.h>
+#include <AzCore/std/string/conversions.h>
+#include <AzCore/std/string/string.h>
+
+namespace AZ
+{
+    class Aabb;
+    class Color;
+    class Matrix3x3;
+    class Matrix4x4;
+    class Quaternion;
+    class Transform;
+    class Vector2;
+    class Vector3;
+    class Vector4;
+}
+
+namespace AZStd
+{
+    //! Prints a Vector2 with precision to 8 decimal points
```

Comment:
Super nitpick - doxygen markup should have a full stop at the end, and I'd suggest "decimal places" rather than "decimal points".